Added Switcher Water Heater component named switcher_kis

TomerFi opened the pull request:
**Description:**
New component for integration with the Switcher V2 Water Heater (https://www.switcher.co.il/).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22325

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

TomerFi committed:
Added Switcher Water Heater component named switcher_kis

klaasnicolaas commented on source/_components/switcher_kis.markdown:
```diff
@@ -0,0 +1,217 @@
+ha_category:
+  - Switch
+  - Sensor
+redirect_from:
```
```suggestion
```

Because this component is new, there are no existing links on the website, so you can omit this.

klaasnicolaas commented on source/_components/switcher_kis.markdown:
```diff
@@ -0,0 +1,217 @@
+---
+
+This `Switcher` component allows you to control the [Switcher V2 Water Heater](https://www.switcher.co.il/).
+
```
```suggestion
There is currently support for the following device types within Home Assistant:
  - Switch
  - Sensor
```

klaasnicolaas committed:
Update source/_components/switcher_kis.markdown

Co-Authored-By: TomerFi <tomer.figenblat@gmail.com>

klaasnicolaas committed:
Update source/_components/switcher_kis.markdown

Co-Authored-By: TomerFi <tomer.figenblat@gmail.com>

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas committed:
:pencil2: Tweak spaces

TomerFi commented:
Thank you @klaasnicolaas 
:-)

klaasnicolaas commented:
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ Pherhaps that the `ha_release` must be updated before merged.

TomerFi commented:
Updated Home Assistant PR to home-assistant/home-assistant/pull/22325.

TomerFi committed:
Fixed typo and canceled the friendly_name configuration value.

TomerFi committed:
Merge branch 'add-switcher_kis-component' of https://github.com/TomerFi/home-assistant.io into add-switcher_kis-component

TomerFi commented:
@klaasnicolaas 
Hi friend, I fixed a typo and canceled the `friendly_name` configuration variable.
I also changed the PR number of the Home Assistant component after changes were made based on the review and change requests.

I will update the `ha_release` value once I'll know exactly which version will this integration will be included in.

TomerFi committed:
Deleted turn_on and turn_off services, added update_schedules service.

TomerFi committed:
Trimmed the component for initial acceptence.

klaasnicolaas commented on source/_components/switcher_kis.markdown:
```diff
@@ -2,30 +2,33 @@
 layout: page
 title: "Switcher"
 description: "Controlling your Switcher V2 Water Heater."
-date: 2018-02-10 18:30
```
It's not necessary to change this every time 😉

TomerFi replied:
LOL I guess you're right!
:-)

andrewsayre commented:
Docs need to be updated to include the scope of the parent PR.

andrewsayre commented on source/_components/switcher_kis.markdown:
```diff
@@ -0,0 +1,87 @@
+  description: The device's password.
+  required: true
+  type: string
+name:
```
Docs need to be updated to match the parent PR

TomerFi replied:
Thank you!
Fixed in this [commit](https://github.com/home-assistant/home-assistant.io/pull/8480/commits/94a81c0bd0189258109757ea1a112f1607056b2d).

TomerFi committed:
Modifications based on recent updates to the homeassistant PR.

andrewsayre commented on source/_components/switcher_kis.markdown:
```diff
@@ -58,25 +53,13 @@ device_password:
-| `last_state_change` | datetime | The datetime of the last change of device's state since the bridge started. | 2019-02-09T13:09:15.005698 |
-| `ip_address` | string | The IP address of the device. | '192.168.0.150' |
-| `device_name` | string | The device's configured name. | "Switcher Boiler" |
+| `friendly_name` | string | The device's configured name. | "Switcher Boiler" |
```
Maybe say this `defaults to the device's configured name`.  If the user customizes it this value, it will be show something different.

TomerFi replied:
Thank you very much!
Fixed in this [commit](https://github.com/home-assistant/home-assistant.io/pull/8480/commits/b6696454c3e1264f42a6da7379168bcd17b273e9).
:-)

TomerFi committed:
Clarifay friendly_name value.

klaasnicolaas commented on source/_components/switcher_kis.markdown:
```diff
@@ -0,0 +1,70 @@
+  - Switch
+redirect_from:
+  - /components/switch.switcher_kis
+ha_release: "0.92"
```
```suggestion
ha_release: 0.92
```

klaasnicolaas commented on source/_components/switcher_kis.markdown:
```diff
@@ -0,0 +1,70 @@
+redirect_from:
+  - /components/switch.switcher_kis
+ha_release: "0.92"
+ha_iot_class: "Local Push"
```
```suggestion
ha_iot_class: Local Push
```

klaasnicolaas committed:
Update source/_components/switcher_kis.markdown

Co-Authored-By: TomerFi <tomer.figenblat@gmail.com>

klaasnicolaas committed:
Update source/_components/switcher_kis.markdown

Co-Authored-By: TomerFi <tomer.figenblat@gmail.com>

andrewsayre committed:
Update ha_release

andrewsayre reviewed (dismissed):
Can merge once build passes.

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas commented:
Some minor adjustments were needed 😉 After this we can merge! 😃
